src/pages/Books.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Book from '../components/Book';
+import Button from '../components/Button';
+import Input from '../components/Input';
+
+function Books() {
+  return (
+    <div className="container">
+      <div className="books-container">
+        <Book title="Book1" author="Alex James" />
+        <div>
+          <Input class="book1" holder="Book title" />
+          <Input class="author" holder="Author" />
+          <Button value="Add Book" />
+        </div>
+      </div>
+      <div className="books-container">
+        <Book title="Book2" author="Marry John" />
+      </div>
+      <div>
+        <Input classes="book" holder="Book title" />
+        <Input classes="author" holder="Author" />
+        <Button value="Add Book" />
+      </div>
+    </div>
```

Comment:
Good job on making separate components for the input fields, But:

- [ ] You are to have a CreateBook form as you will be submitting data to an API in a later section of this milestone and currently, you're rendering the two input fields and button statically. Please create an AddBook/CreateBook component and have a form element there in which you can now use the input field component and a submit button. 👍